src/redux/reducers/componentConfig/initialState.js

```diff
@@ -512,7 +510,6 @@ const mitochondrialFractionLogHistogram = {
   colour: colourBaseState,
   marker: markerBaseState,
   labels: labelBaseState,
-  binStep: 0.05,
```

Comment:
why did you delete these?

Reply:
These binStep removals are related to this discussion https://this-is-biomage.slack.com/archives/C015CHSUDRU/p1643803335465179